src/test/kotlin/lotto/core/YieldCalculatorTest.kt

```diff
@@ -0,0 +1,11 @@
+package lotto.core
+
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+class YieldCalculatorTest {
+    @Test
+    fun `수익률 계산을 검증한다`() {
+        YieldCalculator.calculate(mapOf(WinningRank.RANK4 to 2, WinningRank.RANK3 to 1), 5) shouldBe 12
```

Comment:
테스트 코드도 유지보수의 대상이기 때문에 테스트 코드 또한 어느정도는 읽기 좋은 코드로 유지해야한다고 생각해요. 

꼭 이 양식을 따르라는건 아니지만 테스트 케이스는 given, when, then이 잘 드러나도록 해주시면 
추후 유지보수에도 도움이 될거에요. 😁

```suggestion
        val winningRankCount = mapOf(WinningRank.RANK4 to 2, WinningRank.RANK3 to 1)
        val lottoCount = 5

        val actual = YieldCalculator.calculate(winningRankCount, lottoCount)
        
        actual shouldBe 12
```